packages/credential-provider-ini/src/resolveCredentialSource.ts

```diff
@@ -18,10 +18,9 @@ export const resolveCredentialSource = (
 ): ((options?: CredentialProviderOptions) => Promise<AwsCredentialIdentityProvider>) => {
   const sourceProvidersMap = {
     EcsContainer: async (options?: CredentialProviderOptions) => {
-      const { fromHttp } = await import("@aws-sdk/credential-provider-http");
       const { fromContainerMetadata } = await import("@smithy/credential-provider-imds");
       logger?.debug("@aws-sdk/credential-provider-ini - credential_source is EcsContainer");
-      return chain(fromHttp(options ?? {}), fromContainerMetadata(options));
+      return fromContainerMetadata(options);
```

Comment:
we cannot accept this change, since the `fromHttp` provider implements unique functionality expected by users with a EcsContainer credential_source.

Reply:
Yeah hence my comment around the breaking change (which could be made non-breaking). It was me that asked you for that functionality about two months back (thank you!) so i suspect I'm the only one using it.